packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -20,8 +20,23 @@ import {
     SummaryCollection,
 } from "@fluidframework/container-runtime";
 import { TelemetryNullLogger } from "@fluidframework/common-utils";
-import { IFluidRouter } from "@fluidframework/core-interfaces";
-import { IFluidDataStoreChannel } from "@fluidframework/runtime-definitions";
+import { GenericError } from "@fluidframework/container-utils";
+import { TelemetryDataTag } from "@fluidframework/telemetry-utils";
+
+const getMockStore = ((store: Record<string, string>): Storage => {
+    return {
+        getItem: (key: string): string | null => store[key],
+        length: Object.keys(store).length,
+        clear: () => { },
+        // eslint-disable-next-line no-null/no-null
+        key: (_index: number): string | null => null,
+        removeItem: (_key: string) => { },
+        setItem: (_key: string, _value: string) => { },
+    };
+});
+
+const settings: Record<string, string> = {};
+global.sessionStorage = getMockStore(settings);
```

Comment:
This will go away in the next iteration as it will corrupt the session storage for the while mocha run and risks failing other tests using feature flags. Will be replaced by creating the containers and supplying the config provider to the loader.